control/purchase_order/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class Supplier(models.Model):
+
+    supplier_name = models.CharField(null=False, blank=False, max_length=100)
+    supplier_ref = models.CharField(null=True, blank=True, max_length=100)
+    contact = models.CharField(null=True, blank=True, max_length=100)
+    address = models.CharField(null=True, blank=True, max_length=100)
+    phone = models.IntegerField(null=True, blank=True)
```

Comment:
This is should be a CharField, it could contain different characters such as + or ().

You could also go a bit further and investigate about this: https://stackoverflow.com/questions/19130942/whats-the-best-way-to-store-phone-number-in-django-models